src/main/ui/chat-label.gd

```diff
@@ -42,19 +45,29 @@ func _process(delta: float) -> void:
 
 
 """
-Animates the conversation label to gradually reveal the specified text, mimicking speech.
+Sets the label text, returning 'false' if it causes the label to exceed max_lines_visible
+"""
+func cram_text(text_with_pauses: String) -> bool:
+	_text_with_pauses = text_with_pauses
+	text = text_with_pauses.replace("/", "")
+	return get_line_count() <= max_lines_visible
+
+
+"""
+Animates the chat label to gradually reveal the specified text, mimicking speech.
```

Comment:
"specified text" is confusing, make this comment a little more clear

Reply:
Rephrased to 'current text' to make it clear the text must be previously assigned.

Rephrased remainder of comment as well, as the part about 'returning false' was inaccurate.